docs/markup-language/README.md

```diff
@@ -933,6 +933,36 @@ The amount of federal income tax that must be withheld from the Employee's
 weekly wages is **$[[Amount of Income Tax Withheld]]**.
 ```
 
+## Clauses
+
+The `Clause` type in OpenLaw is equivalent to an embedded template. It is used to insert commonly used legal logic (boilerplate) - whether simple or complex - into existing templates.
+
+For example, assuming you have created a new template called tester, you could reference it in a second template as below:
+
+```
+[[var]]
+[[var 2]]
+[[clause:Clause(name: "tester"; parameters: sub var -> var)]]
+```
+
+Note that since a clause is a type of template, you must create the `tester` template separately before you do the above, or else you will receive the below error:
+
+```
+The template tester could not be found on the server
+```
+
+You can reference the same clause in multiple templates with different parameters. For example, the reference to `tester` clause below will also compile:
+
+```
+<%
+        [[My Variable:Text]]
+        [[Other one:Number]]
+%>
+        
+        [[My Variable]] - [[Other one]]
+        [[_:Clause("tester")]]
+```
```

Comment:
Similar comments for this example. It's not super clear to me how this embedded `Clause` is being used in this different template without that `parameters` prop this time. And if that anonymous variable declaration (`_`) is required to use the `Clause` again?

It might be helpful to show what the end result in the draft view would look like for this example too.

Also, just adding a space after the `:` again and fixing the spacing of each line back to the left.